src/lib.rs

```diff
@@ -15,6 +17,7 @@ use esp_wifi::wifi::{
 };
 use picoserve::response::{IntoResponse, Response};
 
+pub mod mdns;
```

Comment:
I think this module is not defined yet, right? Does the file maybe still need to be committed?

Reply:
Thank you for catching it!